### PR TITLE
#240 chore: implemented secrets through mounting instead of env vars and make sure secret values aren't in the allvalues.yaml

### PR DESCRIPTION
**Pull request by zsinnema**
update the secrets usage from:
```
get_secret("mnp-checkout", "stripe-public-key")
```
to
```
get_secret("stripe-public-key")
```

also no need to do a b64decode anymore
e.g.
```
base64.b64decode(get_secret("mnp-checkout", "stripe-public-key")).decode("ascii")
```
needs to be replaced by
```
get_secret("stripe-public-key")
```

### CODE REVIEW

File: libraries/cloudharness-common/cloudharness/utils/secrets.py [hunk 1]
```diff
@@ -10,4 +10,10 @@ def get_secret(name: str, key: str):
...
         key (str): name of the data key in the secret
     """
-    return os.environ.get(f'{name}-{key}', None)
+    try:
+        secrets_path = '/opt/cloudharness/resources/secrets'
+        with open(os.path.join(secrets_path, name)) as fh:
+            return fh.read()
+    except:
+        # if no secrets folder or file exists
+        return ''
```
Comment (zsinnema) [inline libraries/cloudharness-common/cloudharness/utils/secrets.py:19]: @filippomc or do we here want to raise an exception?

Comment (filippomc) [reply libraries/cloudharness-common/cloudharness/utils/secrets.py:19]: yes definitely this is a runtime exception

**Review by filippomc — CHANGES_REQUESTED** [review]
Integration required: automatically generate helm variable replacements in codefresh pipelines.

Should be something like:


```yaml
  deployment:
    stage: deploy
    image: codefresh/cfstep-helm:3.4.1
    title: Installing chart
    environment:
    ....
    - CUSTOM_apps_[APPNAME]_secrets_github_token=${{GITHUB_TOKEN}}
    - CUSTOM_apps_[APPNAME]_secrets_github_user=${{GITHUB_USER}}
```

**Review by filippomc — APPROVED** [review]
Looks good to me

**Pull request merged**